airflow/dag_processing/processor.py

```diff
@@ -549,6 +549,17 @@ def update_import_errors(session: Session, dagbag: DagBag) -> None:
                     synchronize_session='fetch',
                 )
             else:
+                emails: Set[str] = set(conf.get(section='email', key='dag_import_failure_email_to').split())
+                if emails and conf.getboolean(
+                    section='email', key='email_on_dag_import_failure', fallback=True
+                ):
```

Comment:
```suggestion
                emails: Set[str] = set(conf.get(section='email', key='dag_import_failure_email_to', fallback='').split())
                if emails:
```

Reply:
suggestion taken but wrapped in commit with other changes